kyaml/openapi/openapi.go

```diff
@@ -65,75 +65,77 @@ const SupplementaryOpenAPIFieldName = "openAPI"
 const Definitions = "definitions"
 
 // AddSchemaFromFile reads the file at path and parses the OpenAPI definitions
-// from the field "openAPI"
-func AddSchemaFromFile(path string) error {
-	return AddSchemaFromFileUsingField(path, SupplementaryOpenAPIFieldName)
-}
+// from the field "openAPI", also returns a function to clean the added definitions
```

Comment:
it's a public function (can it be private?) so please explain func and 'clean' more.
```
The returned function is a no-op on error, or else it's a fixed
function that  the caller should use to ... (delete files,
delete junk, whatever "clean" means).
Caller should call the returned func after ...
```